15.md

```diff
@@ -346,24 +339,28 @@ Bids are simply events of kind `1021` with a `content` field containing the stri
 
-The `content` field is a JSON which includes *at least* a `status`. `winner` is how the *winning bid* is replied to after the auction ends and the winning bid is picked by the merchant.
+The `content` field is a JSON object that includes, at a minimum, a `status` field. The `winner` field is used to respond to the winning bid after the auction ends and the merchant has selected the winner.
 
-The reasons for which a bid can be marked as `rejected` or `pending` are up to the merchant's implementation and configuration - they could be anything from basic validation errors (amount too low) to the bidder being blacklisted or to the bidder lacking sufficient *trust*, which could lead to the bid being marked as `pending` until sufficient verification is performed. The difference between the two is that `pending` bids *might* get approved after additional steps are taken by the bidder, whereas `rejected` bids can not be later approved. (NOTE: pending action can happen again after and update of the auction event, the merchant dedices publish an update in the starting bid field of the auction event and can mark as pending bid that have a lower value)
+The reasons for marking a bid as `rejected` or `pending` are determined by the merchant's implementation and configuration. These reasons can range from basic validation errors (e.g., amount too low) to the bidder being blacklisted or lacking sufficient trust, which may lead to the bid being marked as `pending` until additional verification is performed. The key difference between `rejected` and `pending` bids is that `pending` bids may be approved after the bidder takes additional steps, whereas `rejected` bids cannot be later approved. Note that pending bids can be re-evaluated after an update to the auction event, at which point the merchant may publish an update to the starting bid field and mark bids with lower values as pending.
 
-An additional `message` field can appear in the `content` JSON to give further context as of why a bid is `rejected` or `pending`.
+An optional `message` field can appear in the `content` JSON to provide further context for why a bid is `rejected` or `pending`.
 
-Another thing that can happen is - if bids happen very close to the end date of the auction - for the merchant to decide to extend the auction duration for a few more minutes. This is done by passing a `duration_extended` field as part of a bid confirmation, which would contain a number of seconds by which the initial duration is extended. So the actual end date of an auction is always `start_date + duration + (SUM(c.duration_extended) FOR c in all confirmations`.
+If bids are placed very close to the auction's end date, the merchant can extend the auction duration by updating the `duration` field in the auction event.
```

Comment:
Auctions cannot be edited after they have received any bids, so how would this work?

Reply:
There is nothing that stop someone to edit auctions after the first bid, also I'm purposing to do bids differently, instead of using a `e` tag what it is inappropriate to reference p r events (since relays are not supposed to store versions of the p r events) use a 'snapshot' of the auction as stringified auction event in the content and set the amount in a tag `amount` as there is already a standard `amount` tag

Reply:
In the existing NIP, auctions cannot be edited after they get bids. Period. If you edit an auction, it becomes a new product, which has no bids. All bids are attached to a particular version of the auction *by design*.

What you are proposing is to make all clients need to validate this assumption rather than having it simply enforced as part of the protocol. Furthermore, the design gets even more complicated since not only every client needs to verify that the auction did not change, but they should exclude certain fields - like the end time - from the said validation.

Reply:
> relays are not supposed to store versions

There is only one version, because auctions are not supposed to be edited after they receive bids!

Reply:
When editing an auction, whether a bid has been placed or not, it doesn't convert into a new product... This is a parametrizable replaceable event, so forcing clients to behave in a certain way is a solution that raises many potential edge cases. You can't force a user not to change the event, even if clients doesn't allow it, since they can always sign a new version of the same event using their private keys. How do you handle this? You can't. 

What I propose is to use the correct coordinates to point to the auction, using a tag 'a' and storing a stringified version of the current auction event in the bid content (this event is tamper-evident, as it can always be verified that the auctioneer signed the event correctly). This way, the auctioneer can't change anything, and bidders have more security about what they're bidding on. It's also a more robust way to keep track of the auction version being bid on, and use this to put bids on pending or for any other reason. Validating is better than forcing, as forcing is impossible on Nostr, if the user possesses their key pair. The design becomes more robust, not more complicated, by using this snapshot and tag 'a' instead of a tag 'e'. All clients can validate whether a bid is valid or not.

Reply:
>  solution that raises many potential edge cases

Like what? The current protocol and implementation is specifically designed to avoid edge cases because the auction is simply impossible to edit. If you edit it, it becomes a new object with no bids attached. Remember, the event ID is derived from all the event members, so any change to the event means the ID gets changed. This serves as a proof that the bid is attached to a specific version of the auction.

The new proposal seems to introduce a lot of potential issues while solving no real problem.

But I wish other reviewers would also comment, especially since I am not the person deciding whether to merge something or not.